CMakeLists.txt

```diff
@@ -190,6 +190,11 @@ if(SIMENG_ENABLE_TESTS)
 
   endif()
 
+  # Check LLVM version
+  if ((${LLVM_PACKAGE_VERSION} VERSION_LESS "9.0") OR (${LLVM_PACKAGE_VERSION} VERSION_GREATER_EQUAL "13.0"))
+    message(FATAL_ERROR "LLVM version must be >= 9.0 and < 13.0")
```

Comment:
Either "LLVM version must be > "LLVM version must be >= 9.0 and <= 12.0"

Reply:
This would be incorrect as we can support 12.x

Reply:
Comment didn't come through correctly but yes